### DISCUSSION TITLE
Education Verification

### DISCUSSION THREAD

**Discussion by oscardmoreno-UoA** [community/community#84894] [GitHub Education]
### Hi everyone,

Hello,

Why is it so difficult to show proof of education? There is no upload button, and both my pictures from my laptop and iPhone 12 camera come out blurry. My iPhone camera is high quality. What is the problem here? I checked both Firefox and Chrome too. Wasn't the browser. What can I do to verify my application? Once again, why do y'all make this so hard for students? SMH. Please advise/help.

Oscar

**Comment by ANAMASGARD** [comment]
https://youtu.be/yF8s-eIyakY?feature=shared
Watch this video 
Additional tips :-
Camera settings: Make sure your camera settings are on high quality and focus is adjusted.
Check file format: Ensure your pictures are in a supported format like JPG, PNG, or PDF.
Clear browser cache and cookies: Sometimes cached data can cause this problem

**Reply by oscardmoreno-UoA** [reply]
Thanks but for whatever reason my page does not give me an "upload image" option. How did you get it to where there is a button to upload an image?

Thank you,

Oscar
